pkg/ignition/ignition.go

```diff
@@ -132,3 +166,178 @@ func parseKubeletArgs(unitContents string) (map[string]string, error) {
 	}
 	return kubeletArgs, nil
 }
+
+// GetBootstrapFiles returns the paths of all the files needed to bootstrap kubelet
+func (ign *Ignition) GetBootstrapFiles() []string {
+	files := []string{payload.BootstrapKubeconfigPath, payload.KubeletConfigPath, kubeletCACertPath}
+	for _, ignFile := range ign.config.Storage.Files {
+		if ignFile.Path == cloudConfigPath {
+			files = append(files, cloudConfigPath)
+		}
+	}
+	return files
+}
+
+// CreateBootstrapFiles creates all prerequisite files required to start kubelet in the WMCO payload. Returns file paths
+func CreateBootstrapFiles(ign *Ignition, c client.Client, apiServerURL, clusterServiceCIDR string) ([]string, error) {
```

Comment:
Shouldn't this be part of the `Ignition` struct?

Reply:
I wasn't sure since some of the files being created (kubelet config file and bootstrap kubeconfig) don't rely on ignition. I'll reconsider